src/Components/test/testassets/BasicTestApp/GracefulTermination.razor

```diff
@@ -0,0 +1,14 @@
+﻿@inject NavigationManager navigationManager
+
+<h1>Graceful Termination</h1>
+
+<a href="mailto:test@example.com" id="mailto-link">Send Email</a>
+<a href="/subdir/download" download id="download-href">Download Link</a>
```

Comment:
Does it work to simplify this to the following?

```suggestion
<a href="download" download id="download-href">Download Link</a>
```

If so that's slightly preferable as it avoids duplicating the pathbase info. Not a big deal either way though!